src/utils/getFullResUrl.ts

```diff
@@ -0,0 +1,9 @@
+export const GOOGLE_USER_CONTENT_URL = 'https://lh3.googleusercontent.com/';
+const size = 5000;
```

Comment:
Why `5000`? I know we don't really do it anywhere but I am personally a huge fan of explaining 'magic numbers' in code, even with links to source of info.

Reply:
The NFT in question has an original width of 3000 so I figured we could add some headroom but not go too overboard that it manages to crash the app/system.

Reply:
I don't this this a good idea. 5000 means that 3 pixels will be projected to 5 what is not optimal for scaling (eg, pixels will need to be split to 15 (lcm) and then negotiated in groups of 3). Why won't you just drop this ending and let it download the original image without requesting specific width and height?

Reply:
@osdnk It's a max width setting, so only applies to images 5000px wide or over. Google does not allow for downloading the original image without that suffix. 

Check for yourself here:

https://lh3.googleusercontent.com/1Sp7wvdBrflbdGhHrYHx7tcYgUYw5VLq6ETF-SC-yMXkD_t74ybixuP4cHLMtR3ad2EFyTqjMBWXgk8vwFEbs0Mc22JyWj6_bdBjnfU

vs

https://lh3.googleusercontent.com/1Sp7wvdBrflbdGhHrYHx7tcYgUYw5VLq6ETF-SC-yMXkD_t74ybixuP4cHLMtR3ad2EFyTqjMBWXgk8vwFEbs0Mc22JyWj6_bdBjnfU=w5000

Reply:
Ok

Reply:
@osdnk Ahh just found the special argument for rendering the original image buried in some python library. It should be `=s0`. I will get rid of the magic number ✌️